.azure-pipelines/pull-request.yml

```diff
@@ -36,6 +36,17 @@ jobs:
       - template: templates/osx/build-and-unit-test.yml
       - template: templates/osx/pack.unsigned.yml
 
+  - job: linux_build
+    displayName: Linux Build and Unit Test
+    pool:
+      name: ubuntu-latest
+    steps:
+      - powershell: Scripts/CI/Set-Version.ps1 -SourceBranchCounter $(branchCounter)
```

Comment:
Does the PowerShell task #JustWork on linux agents?

Reply:
Almost certainly [not](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/powershell).  :-/  I had a dim recollection that it somehow just worked everywhere from setting up Linux build pipelines for VFS4G, but looking over those again I see we actually used the Bash job in all the Linux pipelines.  Will change, thanks!

Reply:
I don't see it called out but I'm guessing since the `Demand` it makes is the DotNetFramework, that implies Windows? I asked since I know we now support Powershell on mac and linux, but I don't know what that requires :/

Reply:
For now I'm just going to heavily lean on the GitHub Actions setup you created in microsoft/Git-Credential-Manager-Core#157 as a slightly modified version seems to be just fine here.  That simplifies the whole CI setup quite a bit and so for now I think I'll ignore any attempt to get things working in Azure.